app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,48 @@
+<header class="navbar navbar-expand-lg navbar-dark bg-dark">
+  <a class="navbar-brand" href="/">
+    <%= image_tag "Babywearing_Logo.svg", alt:"Mid-Atlantic Babywearing Logo" %>
+  </a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item active background-blue">
+        <%= link_to new_user_registration_path, class: 'nav-link nav-icon' do %>
+          <%= image_tag "BW_Create_User_Icon.svg", size: '50', alt:"Create a User" %>
```

Comment:
@nisusam, noticed that you have an explicit 'size' attribute here and in two places below in addition to having it in the style sheet.  Is there an issue that you're trying to work around?

Reply:
@corprew, In Issue https://github.com/rubyforgood/babywearing/issues/81, @abronkema mentioned
- Adjust sizing of the icons to be uniform

Rails provided `image_tag` helper will take size as a parameter.

https://apidock.com/rails/ActionView/Helpers/AssetTagHelper/image_tag

in which if we passed number `50`, any size image will convert to width="50" and height="50"